content/ui-patterns/messaging.mdx

```diff
@@ -7,25 +7,112 @@ import {Box} from '@primer/react'
 
 Primer includes three different messaging components:
 
-- Toasts
-- [Flash alerts](../components/flash)
-- [Popovers](../components/popover) 
+- Announcement
+- [Banner](../components/flash)
+- [Popover](../components/popover)
+- Toast
+- Callout
+- Inline message
 
-## Messaging types
+## Message types
 
-It's important to consider the context when selecting a component to convey your message. Does the message need to interrupt the user flow? Is this a global message that needs to be shown on every page? Does it require the user to take an action, or is it more of a notice? Most messaging on GitHub falls into one of the following categories:
+Messaging holds differing levels of prominence, which can be used to determine the appropriate component to use. Most messages fall into one of the following categories, from most prominent to least:
 
-**Feedback**: communicates something to the user based on an action they've taken or a change in context
+**System updates**: These types of message originate from GitHub and are not user initiated. A global site wide announcement (outage, missing payment details, critical and blocking) or messaging related to user account/permissions should use the **Announcement** component. System announcements related to a specific product area such as a single sign-on message should use the **Banner** component. System update type messages cannot be dismissed until the issue is resolved, or may be perminent if they are purely informative and not critical.
 
-- _Toasts or flash alerts are best suited for this type of messaging._
+(image here)
 
-**Highlights**: bring attention to how and why to engage with a feature
+**Feedback**: Communicates the result of a user action (submitting a form, toggling a setting, etc.) Feedback can be positive, negative, or a warning. Feedback is shown in either a **Banner** or **Inline message**. For forms, an inline message is used for individual field validation which may be combined with a banner if multiple fields have errors. Banners are placed at the top of the page or section they are related to, but still within the body content. Inline messages are placed near the action they are related to.
 
-- _Popovers are best suited for this type of messaging._
+(image here)
 
-**Updates**: inform the user of necessary updates or changes to the product as it relates to them.
+**Awareness**: Communicates information that is relevant to the user but not necessarily related to an action they've taken. This type of message is typically used to provide context or additional information about a feature or product. A **Callout** might be used to suggest an action or helpful tip, while a **Popover** is typically used to announce a new feature.
```

Comment:
I'm wondering if `Banner` would also fall under this category? It's not totally clear to me what the distinction is between an informative announcement that relates to a specific product area (system update), and something for awareness about a feature or product.

Reply:
Having images here will make it more clear I think, but a Callout is just a less prominent (visually) Banner. But I'll see if I can make the language more clear as well.